contribs/gnodev/README.md

```diff
@@ -2,6 +2,8 @@
 
 `gnodev` is designed to be a robust and user-friendly tool in your realm package development journey, streamlining your workflow and enhancing productivity.
 
+We will only give a quick overview below. You may find the official documentation at [docs/gno-tooling/gnodev.md](../../docs/gno-tooling/cli/gnodev.md).
```

Comment:
Since the urls on docs.gno.land may apparently be refactored soon as per #2262, I suggest to keep a relative link here for now.

Reply:
We generally use local relateve links in the docs